contents/docs/libraries/node/index.mdx

```diff
@@ -124,6 +124,33 @@ import NodeFeatureFlagsCode from '../../integrate/feature-flags-code/_snippets/f
 
 import LocalEvaluationIntro from "../../feature-flags/snippets/local-evaluation-intro.mdx"
 
+
+### GeoIP Property evaluation
+
+If your feature flag depends on geoIP properties, like the country or city, you can ask us to automatically override these properties based on the current IP address.
```

Comment:
Why would they want to "automatically override these properties"? As opposed to just doing whatever the default behaviour is? What is the default behviour?

Reply:
Default behaviour is to use whatever property is on the person.

Buuut the frontend by default overrides it based on the current IP address, because that makes the most sense - your current IP = more current geoIP information, vs whatever is stored on the person.

So, for most co's, the default is what they want.

But, for some, there might be no frontend, and their backend is the only place that is flagged, and they want to restrict API requests based on the IP, so there they might want to forward their IP to us, and us the properties from this IP to determine whether to enable or disable flags

Reply:
Okay thanks, I think I get it now. 

Just to confirm, in the context of Node/Python servers:
- Default behaviour is to use the user property values for geolocation
- The properties are set from the frontend for geolocation
- However, if there is no frontend user, then there are no geolocation properties for that user
- Thus, to use geolocation properties from the server, use `disableGeoip: false`. This will set the properties for the user (or event?) based on the servers IP
     - If they dont want to use the server IP, they can pass through `HTTP_X_FORWARDED_FOR` or `REMOTE_ADDR` to use the user's IP. How would would they do?

Reply:
> How would would they do this?

Good question, looks like they can't currently with the SDKs, we don't allow adding extra headers 🤦

Reply:
Makes sense 👍

Reply:
If you like, Im happy to take over from here and implement the suggestions?

Reply:
thank you, whenever works!